src/app/components/footer/footer.component.html

```diff
@@ -1,88 +1,85 @@
-                </div>
-
-            </div>
-            <!-- Grid column -->
-
-            <hr class="clearfix w-100 d-md-none pb-3">
-
-            <!-- Grid column -->
-            <div class="col-md-6 mb-md-0">
-
-                <!-- Links -->
-                <div class="mb-5 flex-center">
-
-                    <!-- Facebook -->
-                    <a class="fb-ic">
-                        <i class="fab fa-facebook-f fa-lg white-text mr-md-5 mr-3 fa-2x"> </i>
-                    </a>
-                    <!-- Twitter -->
-                    <a class="tw-ic">
-                        <i class="fab fa-twitter fa-lg white-text mr-md-5 mr-3 fa-2x"> </i>
-                    </a>
-                    <!-- Google +-->
-                    <a class="gplus-ic">
-                        <i class="fab fa-google-plus-g fa-lg white-text mr-md-5 mr-3 fa-2x"> </i>
-                    </a>
-                    <!--Linkedin -->
-                    <a class="li-ic">
-                        <i class="fab fa-linkedin-in fa-lg white-text mr-md-5 mr-3 fa-2x"> </i>
-                    </a>
-                </div>
-
-            </div>
-            <!-- Grid column -->
-            <hr class="clearfix w-100 d-md-none pb-3">
-            <!-- Grid column -->
-            <div class="col-md-3 mb-md-0 mb-3">
-
-                <!-- Links -->
-                <h5 class="text-uppercase">Contact us</h5>
-
-                <ul class="list-unstyled">
-                    <li>
-                        8, Lasode Cresent, Victoria Island <br/>
-                        Lagos, Nigeria
-                    </li>
-                    <li>
-                        07123456789
-                    </li>
-                    <li>
-                        www.thelandville.com
-                    </li>
-                    <li>
-                        info@thelandville.com
-                    </li>
-                </ul>
-
-            </div>
-            <!-- Grid column -->
-
+  <!-- Footer Links -->
+  <div class="container-fluid text-center text-md-left">
+    <!-- Grid row -->
+    <div class="row">
+      <!-- Grid column -->
+      <div class="col-md-3 mt-md-0 mt-3">
+        <!-- Content -->
+        <div class="footer-widget d-flex justify-content-center">
+          <img
+            alt=""
+            class="img-fluid"
+            height="150"
+            src="assets/img/ICON/Logo Icon.png"
+            width="150"
+          />
         </div>
-        <!-- Grid row -->
-
-    </div>
-    <!-- Footer Links -->
-    <hr class="clearfix w-100 d-md-none pb-3">
-    <!-- Copyright -->
-    <div class="footer-copyright text-center py-3">© {{ currentDate | date: 'yyyy' }} Copyright:
-        <a routerLink="/"> Landville</a>
+      </div>
+      <!-- Grid column -->
+
+      <hr class="clearfix w-100 d-md-none pb-3" />
+
+      <!-- Grid column -->
+      <div class="col-md-6 mb-md-0">
+        <!-- Links -->
```

Comment:
leave the comments for the social links but most of the other comments in html are redundant to my thinking.

Reply:
Thanks @kdaglas , I have worked on this too!